dev/com.ibm.ws.org.jboss.resteasy.resteasy.servlet.initializer/bnd.bnd

```diff
@@ -0,0 +1,35 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= jar:${fileuri;${repo;org.jboss.resteasy:resteasy-servlet-initializer;4.5.2.Final;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
+# bVerison goes in overrides
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8\
+
+
+Include-Resource:\
+  @${repo;org.jboss.resteasy:resteasy-servlet-initializer;4.5.2.Final;EXACT}
+#  org/jboss/resteasy=${bin}/org/jboss/resteasy
+
+-buildpath: \
+	org.jboss.resteasy:resteasy-servlet-initializer;version=4.5.2.Final,\
+	com.ibm.ws.org.jboss.resteasy.resteasy.core,\
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+	com.ibm.websphere.org.osgi.service.component;version=latest,\
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+	com.ibm.websphere.javaee.jaxrs.2.1,\
+	com.ibm.websphere.javaee.servlet.4.0,\
+	com.ibm.ws.org.reactivestreams.reactive-streams.1.0,\
```

Comment:
does the servlet initializer really need reactive streams?  seems like an odd dependency...

Reply:
I needed it when I was including the source code, I'll remove it.